examples/wifi-echo/server/esp32/main/wifi-echo.cpp

```diff
@@ -63,15 +63,13 @@ extern void startClient(void);
 #define BUTTON_2_GPIO_NUM GPIO_NUM_38               // Middle button on M5Stack
 #define BUTTON_3_GPIO_NUM GPIO_NUM_37               // Right button on M5Stack
 #define STATUS_LED_GPIO_NUM GPIO_NUM_MAX            // No status LED on M5Stack
-#define LIGHT_CONTROLLER_OUTPUT_GPIO_NUM GPIO_NUM_2 // Use GPIO2 as the light controller output on M5Stack
 
 #elif CONFIG_DEVICE_TYPE_ESP32_DEVKITC
 
 #define BUTTON_1_GPIO_NUM GPIO_NUM_34                // Button 1 on DevKitC
 #define BUTTON_2_GPIO_NUM GPIO_NUM_35                // Button 2 on DevKitC
 #define BUTTON_3_GPIO_NUM GPIO_NUM_0                 // Button 3 on DevKitC
 #define STATUS_LED_GPIO_NUM GPIO_NUM_2               // Use LED1 (blue LED) as status LED on DevKitC
```

Comment:
Actually, as per the code, this `STATUS_LED_GPIO_NUM` is used for on/off/toggle commands from the CHIPTool. If that's the right thing, should this be renamed to `LIGHT_CONTROLLER_OUTPUT_GPIO_NUM` ?